Software/Python/ir_remote_control/lirc/install.sh

```diff
@@ -45,49 +45,15 @@ then
 	echo " "
```

Comment:
Where is this install called from when doing a Raspbian for Robots update?  How is it automatically called?